subprojects/docs/src/docs/release/notes.md

```diff
@@ -117,6 +117,10 @@ repositories {
 
 In previous versions of Gradle the URL referred to by `RepositoryHandler.mavenCentral()` was pointing to `https://repo1.maven.org/maven2/`. Sonatype recommends using the canonical URL `https://repo.maven.apache.org/maven2/` instead. This version of Gradle makes the switch to `repo.maven.apache.org` when using the `mavenCentral()` API.
 
+### Provider API documentation
+
+In this release, the Gradle team added a new chapter in the user guide documenting the [Provider API](userguide/lazy_configuration.html).
+
```

Comment:
Leave this here, but I think it would be good to incorporate this into the preamble that @eriwen is writing.

Reply:
Yes, yes, yes.